frontend/public/components/cluster-settings/cluster-operator.tsx

```diff
@@ -49,7 +49,7 @@ export const getClusterOperatorStatus = (operator: K8sResourceKind) => {
 const getIconClass = (status: OperatorStatus) => {
   return {
     [OperatorStatus.Available]: 'pficon pficon-ok text-success',
-    [OperatorStatus.Updating]: 'pficon pficon-in-progress',
+    [OperatorStatus.Updating]: 'fa-spin pficon pficon-in-progress',
```

Comment:
I would avoid animating these so we don't have 40 spinning icons on upgrade. (They can get out of sync too if they start upgrading at slightly different times.)